tests/FunctionalJavascript/TranslationTestBase.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_translation\FunctionalJavascript;
+
+use Drupal\FunctionalJavascriptTests\WebDriverTestBase;
+
+/**
+ * Base class for functional javsacript tests.
+ */
+class TranslationTestBase extends WebDriverTestBase {
```

Comment:
We don't use this base class anywhere. Also it should probably be abstract (if we keep it)

Reply:
We will use it soon so let's not delete it. Are the other base test classes abstract? If so, yes, make them abstract

Reply:
The other base tests are not abstract neither, even though I think they should be. Can be fixed at a later stage.